DigitalLearningSolutions.Web/Views/Shared/Components/DateInput/Default.cshtml

```diff
@@ -13,10 +13,12 @@
     <legend class="nhsuk-fieldset__legend nhsuk-label">
       @Model.Label
     </legend>
-    @if (Model.HintText != null) {
-      <div class="nhsuk-hint" id="@Model.Id-hint">
-        @Model.HintText
-      </div>
+    @if (Model.HintTextLines != null) {
+      @foreach (var hintText in Model.HintTextLines) {
+        <div class="nhsuk-hint">
+          @hintText
+        </div>
+      }

```

Comment:
Can't currently retest this because of the new migration changes, but I thought that the spacing was too close between the hint text and the input, but I can't currently confirm/deny if I messed something up at the time. Might double check later.

Reply:
Here's an example of the date input with multiple hint lines in the tracking system, and the NHS example of multiple hint lines, I think they look the same?
![localhost_5001_TrackingSystem_Delegates_ViewDelegate_DelegateProgress_284958_EditCompleteByDate (4)](https://user-images.githubusercontent.com/70278044/146235819-b63641dc-42fe-4487-9297-af6490f2fd53.png)
![service-manual nhs uk_design-system_components_hint-text (1)](https://user-images.githubusercontent.com/70278044/146236215-0d813aa1-1732-41ae-a5bb-21b5c6adb304.png)